torch/csrc/jit/mobile/file_format.h

```diff
@@ -159,7 +153,6 @@ static inline std::tuple<std::shared_ptr<char>, size_t> get_file_content(
   return std::make_tuple(data, size);
 }
 
-// NOLINTNEXTLINE(facebook-hte-NamespaceScopedStaticDeclaration)
```

Comment:
Ugh, I think these are internal clang-tidy checks? We can't  shouldn't just disable them.